webpack.config.js

```diff
@@ -19,20 +19,20 @@ var commonConfig = {
 
   output: {
     path:       outputPath,
-    filename: `/static/js/${outputFilename}`,
-    // publicPath: '/'
+    filename:   `static/js/${outputFilename}`,
+    publicPath: '/'
   },
 
   resolve: {
-    extensions: ['', '.js', '.elm']
+    extensions: ['.js', '.elm']
   },
 
   module: {
     noParse: /\.elm$/,
-    loaders: [
+    rules: [
       {
         test: /\.(eot|ttf|woff|woff2|svg)$/,
-        loader: 'file-loader'
+        use:  'file-loader'
```

Comment:
I think this should still be `loader`. Check the `// Do not use "use" here` comment in the code block at https://webpack.js.org/guides/migrating/#module-loaders-is-now-module-rules